requirements/pip.txt

```diff
@@ -96,7 +96,7 @@ packaging==19.0
 django-cors-middleware==1.3.1
 
 # User agent parsing - used for analytics purposes
-user-agents<1.2.0
+user-agents==2.0
```

Comment:
From the changelog

> python-user-agents now require ua-parser>=0.8.0

https://github.com/selwin/python-user-agents#version-20-2019-04-07